modules/nw-osp-pod-connections-ipv6.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-connections-ipv6_{context}"]
+= Enabling IPv6 connectivity to pods on {rh-openstack}
+
+To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security obviates the need to create allowed address pairs.
```

Comment:
```suggestion
To enable IPv6 connectivity between pods that have additional network that are on different nodes, the port security for the IPv6 port of the server can be disabled. Disabling port security obviates the need to create allowed address pairs for each IPv6 address assigned to Pods and enable traffic on the security group.
```

Reply:
Some questions about the suggestion. 

1. Is only one additional network allowed per pod? 
2. Must the IPv6 port be disabled, or is it optional? 

At the moment, the PR assumes that the answer to 1 is "no" and the answer to 2 is "yes."

@MaysaMacedo

Reply:
1. there can be more additional networks.
2. It was tested with disabling the security to facilitate the user experience. Otherwise the user would need to add one allowed address pair with the ipv6 address of the pod and enable the security group port (AFAIK this other option was not tested).

Reply:
@MaysaMacedo Given those, I'd edit your suggestion like so:

```suggestion
To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security obviates the need to create allowed address pairs for each IPv6 address that is assigned to pods and enable traffic on the security group.
```
Keeps "networks" plural and states unambiguously that this is the official way to accomplish this.